earth_enterprise/src/server/geresetpgdb

```diff
@@ -181,15 +181,21 @@ function do_dump {
 
 # Deletes old data and recreates db instance.
 function do_reset {
-  if [ -f $VARPGDIR/data/postmaster.pid ]; then
-    $BINDIR/pg_ctl -D $VARPGDIR/data -l $VARPGDIR/logs/pg.log stop
-  fi
 
-  # delete all the db files. not sure if we should do this.
-  rm -Rf $VARPGDIR/data
+  # Drop gee databases
+  if [ `$BINDIR/psql -lqt | cut -d \| -f 1 | grep -w gestream | wc -l` == 1 ]; then
```

Comment:
If I list databases, I see the following:

```
 geendsnippet  | gepguser | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 geplaces      | geuser   | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 gepoi         | gepguser | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 gesearch      | gepguser | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 gestream      | gepguser | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 postgres      | gepguser | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 searchexample | geuser   | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 template0     | gepguser | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/gepguser
                                                                 : gepguser=CTc/
gepguser
 template1     | gepguser | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/gepguser
                                                                 : gepguser=CTc/
```

There are apparently other databases that should be dropped.  Hopefully we aren't missing any that are created dynamically.

Reply:
This is the list of dbs that were always dropped in the "soft" reset. geplaces and searchexample are only dropped in the "hard" reset. I just moved it around so that the do_reset function is called in both soft and hard reset, but the example databases are only dropped for hard.